test/e2e/integration_test.go

```diff
@@ -43,19 +43,19 @@ var _ = Describe("When testing integration [integration]", func() {
 		By("Deprovision target cluster")
 		bootstrapClient := bootstrapClusterProxy.GetClient()
 		intervals := e2eConfig.GetIntervals(specName, "wait-deprovision-cluster")
-		// In pivoting step we labeled the BMO CRDs (so that the objects are moved 
+		// In pivoting step we labeled the BMO CRDs (so that the objects are moved
 		// by CAPI pivoting feature), which made CAPI DeleteClusterAndWait()
 		// fail as it has a check to make sure all resources managed by CAPI
 		// is gone after Cluster deletion. Therefore, we opted not to use
-		// DeleteClusterAndWait(), but only delete the cluster and then wait 
+		// DeleteClusterAndWait(), but only delete the cluster and then wait
 		// for it to be deleted.
 		framework.DeleteCluster(ctx, framework.DeleteClusterInput{
 			Deleter: bootstrapClient,
 			Cluster: result.Cluster,
 		})
 		Logf("Waiting for the Cluster object to be deleted")
 		framework.WaitForClusterDeleted(ctx, framework.WaitForClusterDeletedInput{
-			Getter: bootstrapClient,
+			Getter:  bootstrapClient,
```

Comment:
extra space

Reply:
This is from `go fmt`, it aligns with the line below so it is easier to read. Go fmt would not accept this with one less space because then it would not follow the go fmt style.

Reply:
Weird! Next line seems only one space !

Reply:
Oh I see format the values on the same line